.travis.yml

```diff
@@ -37,9 +37,13 @@ addons:
     - r-source
     - sourceline: 'deb http://cran.rstudio.com/bin/linux/ubuntu trusty/'
       key_url: 'keyserver.ubuntu.com/pks/lookup?op=get&search=0x51716619E084DAB9'
+    - r-packages-trusty
+    - mysql-5.7-trusty
     packages:
     - r-base
     - r-base-dev
+    - mysql-server
+    - mysql-client
```

Comment:
why is this needed?

Reply:
This is for `JdbcIntegrationTest`, otherwise this test will fail